src/uu/dmesg/src/dmesg.rs

```diff
@@ -49,16 +66,27 @@ pub fn uu_app() -> Command {
                 .help("use JSON output format")
                 .action(ArgAction::SetTrue),
         )
+        .arg(
+            Arg::new(options::TIME_FORMAT)
+                .long("time-format")
+                .help(
+                    "show timestamp using the given format:\n".to_string()
+                        + "  [delta|reltime|ctime|notime|iso|raw]",
+                )
+                .action(ArgAction::Set),
+        )
```

Comment:
Did you deliberately leave out the related arguments `--notime`, `--ctime` and `--reltime` for a future PR?

Reply:
Yes, I am planning on creating a separate PR for handling multiple time formats (e.g. `dmesg --ctime --reltime --time-format delta ...`). Since if I add the short hand options now, I would need to temporarily decide the precedence of those options.

The reason why I want to create the multiple time format handling in a separate PR is because of how peculiar the tool seems to handle `delta` time format when it is specified together with other time formats.

What do you think?

Reply:
It makes sense.